index.js

```diff
@@ -0,0 +1,237 @@
+    {but: 'ю', keyCode:190,code:'Period'},{but: '.', keyCode:191,code:'Slash'},{but: '▲', keyCode:38,code:'ArrowUp'},{but: 'Shift', keyCode:16,code:'ShiftRight'},
+    {but: 'Ctrl', keyCode:17,code:'ControlLeft'},{but: 'Win', keyCode:91,code:'MetaLeft'},{but: 'Alt', keyCode:18,code:'AltLeft'},
+    {but: 'space', keyCode:32,code:'Space'},{but: 'Alt', keyCode:18,code:'AltRight'},{but: '◄', keyCode:37,code:'ArrowLeft'},{but: '▼', keyCode:40,code:'ArrowDown'},
+    {but: '►', keyCode:39,code:'ArrowRight'},{but: 'Ctrl', keyCode:17,code:'ControlRight'}
+];
+const Keyboard = {
+    
+    elements:{
+        main: null,
+        keysContainer: null,
+        keys: []
+    },
+
+    evenstHandlers:{
+        oninput:null,
+    },
+
+    properties:{
+        value:'',
+        capsLock:false
+    },
+
+    init() {
+        this.elements.textarea = document.createElement('textarea');
+        this.elements.main = document.createElement('div');
+        this.elements.keysContainer = document.createElement('div');
+        this.elements.wrap = document.createElement('div');
+        this.elements.wrap.className = 'wrapper';
+        this.elements.main.className = 'keyboard';
+        this.elements.textarea.className = 'use-keyboard-input';
+        this.elements.keysContainer.className = 'keyboard__keys';
+        this.elements.main.prepend(this.elements.keysContainer);
+        document.body.prepend(this.elements.wrap);
+        this.elements.wrap.append(this.elements.main);
+        this.elements.wrap.prepend(this.elements.textarea);
+        this.elements.keysContainer.appendChild(this._createKeys());
+        this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard__key');
+     
+        document.querySelectorAll('.use-keyboard-input').forEach(element=>{
+            element.addEventListener('focus',()=>{
+                this.open(element.value, currentValue=>{
+                    element.value = currentValue;
+                });
+                
+            });
+        });
+    },
+
+    _createKeys() {
+        const fragment = document.createElement('div');
+        // const keyLayoutEng = [
+        //     '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+        //     'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'DEL',
+        //     'CapsLock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', 'ENTER',
+        //     'shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', 'top', 'Shift',
+        //     'Ctrl', 'Win', 'Alt', 'space', 'Alt', 'left', 'bottom', 'right', 'Ctrl'
+        // ];
+        const createIconHtml = (icon_name)=>{
+            return `<i class='material-icons'>${icon_name}</i>`;
+        };
+        for(let key in keyLayout){
+            const keyElement = document.createElement('button');
+            const insertLineBreak = ['Backspace', 'Delete', 'Enter','ShiftRight'].indexOf(keyLayout[key].code) !== -1;

```

Comment:
It is difficult to understand what this code is for. It is better to use named variables or configs